static-build/components/TestCard/styles.css

```diff
@@ -17,7 +17,7 @@
     justify-content: center;
     width: 100%;
     transition: 0.2s ease-in-out;
-    will-change: box-shadow, transform;
```

Comment:
I think you can keep will-change:transform because the transform really will change. This can optimize animation performance, and reduce CPU usage and possible hiccup on start/end of the animation.